vent/coordinator/rpc.py

```diff
@@ -1,11 +1,14 @@
 import pickle
+import socket
 import xmlrpc.client
 from xmlrpc.server import SimpleXMLRPCServer
 
 import vent.controller.control_module
 
 default_addr = 'localhost'
 default_port = 9533
+default_timeout = 10
```

Comment:
Is this 10 seconds just for the sake of being something, or did that number come from somewhere? 10s seems kind of high to me

Reply:
before it was infinity, i figure we should pull these kinds of prefs out to a separate file so the user can set.